vm/src/pyobject.rs

```diff
@@ -512,21 +512,30 @@ impl fmt::Debug for PyObject {
 #[derive(Debug, Default, Clone)]
 pub struct PyFuncArgs {
     pub args: Vec<PyObjectRef>,
-    // TODO: add kwargs here
+    pub kwargs: Option<Vec<(String, PyObjectRef)>>,
```

Comment:
Finally :+1:

Reply:
Btw, is option required? An empty vector means no kwargs?

Reply:
Nope, not required; just pushed up a fix.